specutils/io/default_loaders/ecsv.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from astropy.table import Table
+
+from specutils.io.registers import data_loader, custom_writer
+from specutils import Spectrum1D
+
+__all__ = ['ecsv_identify', 'ecsv_spectrum_loader']
+
+
+def ecsv_identify(origin, *args, **kwargs):
+    """Check if it's an ECSV file."""
+    name = os.path.basename(args[0])
+
+    if name.lower().split('.')[-1] == 'ecsv':
+       return True
+
+    return False
+
+
+@data_loader(label="Spectrum ECSV", identifier=ecsv_identify, extensions=['ecsv'])
+def ecsv_spectrum_loader(file_name, **kwargs):
+    """
+    Load spectrum from ECSV file
```

Comment:
I think this is much safer than the "generic ASCII" because it's unambiguous what the names of the table columns are.  *however*, the docstring needs to be much more specific.  It should say what column names it's expecting in the table.